tests/test_deploy_github_pages.py

```diff
@@ -571,7 +589,7 @@ def test_use_different_source_dir(setup_test_env):
         "--source-dir", "documentation/",
         "--module-path", "../test_package",
         "--module-path", "../another_test_package",
-        "--debug"]
+        "--push"]
```

Comment:
why did this change?

Reply:
the debug was a remainder from a debugging session and is not needed, and the push was added because i changed the default from True to False.